src/main/features/core/tray.js

```diff
@@ -180,5 +180,50 @@ Emitter.on('audiooutput:list', (event, devices) => {
   setContextMenu();
 });
 
+// Change the Icon if its playing
+Emitter.on('playback:isPlaying', () => {
+  if (process.platform === 'darwin') {
+    appIcon.setImage(path.resolve(`${__dirname}/../../../assets/img/playing/macTemplate.png`));
```

Comment:
Nonono, I meant constants for the filename not the root path.  Then we only need to do the massive if chain once, not multiple times